acquia_lift.module

```diff
@@ -3063,6 +3136,45 @@ function acquia_lift_get_campaign_details() {
 }
 
 /**
+ * Determines if the Lift debugger should be enabled on the page request.
+ *
+ * @return bool
+ *   TRUE to include on the page, FALSE otherwise.
+ */
+function acquia_lift_debug_mode_enabled() {
+  $enabled = personalize_debug_mode_enabled();
+  if (!$enabled) {
+    return FALSE;
+  }
+  if (!acquia_lift_is_configured()) {
+    return FALSE;
+  }
+  // Check to see if there is a session variable for debug mode.
+  $in_session = isset($_SESSION['acquia_lift_inspect_mode']);
```

Comment:
Darn - we should set the 'acquia_lift_inspect_mode' url parameters as a constant and just refer to that.